tests/routes/test_analysis.py

```diff
@@ -1,9 +1,37 @@
 import pytest
+import json
+import asyncio
 
 from app.errors import InvalidResponseError
 from app.settings.globals import (
-    RASTER_ANALYSIS_LAMBDA_NAME,
+    RASTER_ANALYSIS_LAMBDA_NAME
 )
+from app.utils.aws import get_lambda_client
+import app.routes.analysis.analysis as analysis
+
+
+# Workaroud because of this bug with aiobotocore: https://github.com/aio-libs/aiobotocore/issues/755
+# It's not working correctly with moto, so need to monkeypatch the lambda invoke to be synchronous
+# during tests
+@pytest.yield_fixture(scope='session')
+def event_loop(request):
+    loop = asyncio.get_event_loop_policy().new_event_loop()
+    yield loop
+    loop.close()
+
+
+@pytest.fixture(autouse=True)
+def patch_invoke(monkeypatch):
+    async def mock_invoke(payload):
+        response = get_lambda_client().invoke(
+            FunctionName=RASTER_ANALYSIS_LAMBDA_NAME,
+            InvocationType="RequestResponse",
+            Payload=bytes(json.dumps(payload), "utf-8"),
+        )
+
+        return response['Payload'].read()
+
+    monkeypatch.setattr(analysis, "_invoke_lambda", mock_invoke)
```

Comment:
Did you look into the moto server option?
Hope we find some time at one point to investigate localstack as an alternative.

Reply:
Yeah, I couldn't make the lambda server work correctly because it depended on shared state with the IAM service (for assuming the lambda role), but you can't run multiple services on the same moto server. Localstack does support that, and that's what people online recommended. It didn't seem worth it to move everything to Localstack just for this small change, but I'm in favor of looking into this soon

Reply:
Actually I'll just quickly look now if I can set it up just for this test suite, and we can migrate the rest of the tests later

Reply:
Hmm, for some reason aioboto3 only runs on a very pretty version of boto3 (1.12.32) which localstack doesn't support (>=1.14.33). So still might not work here, although localstack is pretty easy to set up, so we should definitely put some time in to switch over soon

Reply:
ok, thanks for investigating.
Let's work with Liza to put some time aside to improve the test setup.

Reply:
Just another thought. What happens if you only use aiobotocore? 
There would be less dependencies on external libraries. Would you still have botocore version restrictions?
Or do you need boto3 to use moto server/ localstack?

Reply:
That actually works if I downgrade localstack a little. Although I realized I really only need to run it in a docker container, the library itself is in unnecessary on the server. But I still can't seem to get it to work, it just keeps giving me weird and undescriptive errors when it runs my test lambda that moto had no issue with it. 

If I can't figure it out by Monday, can we just go with this test and move to localstack when we set it all up? I'd like to get OTF moving towards production.

Reply:
Thanks for looking into this. And yes, I am fine with putting this on the backburner.